bugbug/models/devdocneeded.py

```diff
@@ -15,6 +15,7 @@
 
 class DevDocNeededModel(Model):
     def __init__(self, lemmatization=False):
+        self.undersampling_enabled = True
```

Comment:
Remove this, and add sampler = RandomUnderSampler (after the Model.__init__ call)

Reply:
Same for qaneeded and uplift